docs/Reference/TypeScript.md

```diff
@@ -661,6 +661,21 @@ However, there are a couple of suggestions to help improve this experience:
 - Make sure the `no-unused-vars` rule is enabled in
   [ESLint](https://eslint.org/docs/rules/no-unused-vars) and any imported plugin
   are actually being loaded.
+- In case you've the `@typescript-eslint/no-floating-promises` enabled,
+please double-check that your `.eslintrc` includes `allowForKnownSafePromises`
+property as described [here](https://github.com/fastify/fastify/issues/5498):
```

Comment:
Can we point to the official docs instead?

Reply:
Done on [52c9dab](https://github.com/fastify/fastify/pull/5506/commits/52c9dab1eb4537752aa882acebbbb8187c2c6c9b)